### DISCUSSION TITLE
Phishing sites - how to report

### DISCUSSION THREAD

**Discussion by genman** [community/community#70329] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Fake site:
https://we-apple-customer-reach-us.github.io/care/?utm_medium=marketing&_branch_referrer=H4sIAAAAAAAAAxXKUQ9CQADA8U%2BjR6w0arN2DjFqqofKS0PidDh3p%2Bv69LH9n%2F771ZwTttU061W3hpoRomLUvbWDLG8Mp%2FTu5ruRYruembICytKfYiMhPeUzx6Va9O30FlWB0dOGjR7BUyOMHBGnhhKcQwZ8hkr4G4fAN0VjJkiaa7oy3GbfkkDPclJdNjREJRadx5NvFA%2BV%2FKRXVx6tDHDP8kAsHqJw%2FjGCNCusAAAA&url=https%3A%2F%2Fsupport.apple.com%2F&gclid=Cj0KCQjw4bipBhCyARIsAFsieCzuqHF7wj7Piy75r34DjGmpH0abpgS9rIielwnEtPxKLqgyvZWDyN8aAtE8EALw_wcB&_branch_match_id=1205936641237082101

Link leads you to a tech support scammer number: +1-(844)-781-6134

From:
https://www.reddit.com/r/ScamNumbers/comments/17a29nl/apple_support_scam_works_as_of_101723_1225pm_est/

How is there no easy way to report this?

**Comment by queenofcorgis** [comment, marked as answer]
Hi @genman 👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
